src/tribol/geom/GeomUtilities.cpp

```diff
@@ -818,7 +812,7 @@ void Intersection2DPolygon( const real* const RESTRICT xA,
          // debug
          if (k > (numPolyVert))
          {
-            TRIBOL_ERROR("Intersection2DPolygon: iterator k greater than numPolyVert (1)");
+            SLIC_DEBUG("Intersection2DPolygon: iterator k greater than numPolyVert (1)");
```

Comment:
Would more context be useful for debugging this?

Reply:
Also, `SLIC_DEBUG_IF( <cond>, <msg>)` might be more concise

Reply:
Fixed.